Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3-alpine
+MAINTAINER Tecnativa <info@tecnativa.com>
+ENTRYPOINT ["/usr/src/app/entrypoint.sh"]
+ENV GIT_AUTHOR_NAME=git-aggregator \
+    EMAIL=https://hub.docker.com/r/tecnativa/git-aggregator
```

Comment:
This is because this code builds https://hub.docker.com/r/tecnativa/git-aggregator/, and I don't want to address people to Acsone until they acknowledge maintenance of this image (in which case it would become https://hub.docker.com/r/acsone/git-aggregator/ or something similar). That's why I said what I said in https://github.com/acsone/git-aggregator/pull/5#issue-197990467.